src/github/client/github-client.ts

```diff
@@ -26,5 +30,42 @@ export class GitHubClient {
 			this.restApiUrl = `${baseUrl}/api/v3`;
 			this.graphqlUrl = `${baseUrl}/api/graphql`;
 		}
+
+		this.axios = axios.create({
+			baseURL: this.restApiUrl,
+			transitional: {
+				clarifyTimeoutError: true
+			},
+			... this.getProxyConfig(this.restApiUrl)
+		});
 	}
+
+	public getProxyConfig = (baseUrl: string): Partial<AxiosRequestConfig> => {
+		if (baseUrl
+			.replace(RegExp("/$"), "")
```

Comment:
I'm not sure this is going to work as you expect it to.  GHE needs to have `/api/` in baseUrl. Instead, I'd probably do something like `new URL(baseUrl).host.endsWith("atlassian.com")` .  Do we also want to include atlassian.net?

Reply:
good catch. Although, it worked in my tests. 

I made it more robust now ... just atlassian.com for now because I hope this is the only case we'll ever need (before we rip it out again)